spring-web-modules/spring-mvc-basics-4/src/main/java/com/baeldung/contexts/secure/HelloWorldSecureController.java

```diff
@@ -31,19 +30,15 @@ private void processContext() {
         ApplicationContext context = contextUtilService.getApplicationContext();
         System.out.println("application context : " + context);
         System.out.println("application context Beans: " + Arrays.asList(context.getBeanDefinitionNames()));
-        
-        WebApplicationContext rootContext = ContextLoader.getCurrentWebApplicationContext();
-        System.out.println("context : " + rootContext);
-        System.out.println("context Beans: " + Arrays.asList(rootContext.getBeanDefinitionNames()));
 
         System.out.println("context : " + webApplicationContext);
         System.out.println("context Beans: " + Arrays.asList(webApplicationContext.getBeanDefinitionNames()));
     }
 
-    @GetMapping(path = "/welcome")
+    @GetMapping(path = "/welcome_secure")
```

Comment:
why are we changing the endpoint here? does it makes any difference?

Reply:
Because we have already a `welcome` path in the `HelloWorldController`.